htgr/httf/sam_ring_model/HTTF-SS.i

```diff
@@ -0,0 +1,6651 @@
+    0.000049388 0.000050022 0.000050654 0.000051282 0.000051907 0.00005253 0.000053149 0.000053766 0.00005438 0.000054991 0.0000556 0.000056206 0.000056809 0.00005741 0.000058009 0.000058605 0.000059198 0.00005979 0.000060379 0.000060966 0.000061551'
+  []
+
+  [kHe] #Helium therm. cond; x- Temperature [K], y-Thermal condiuctivity [W/m-K]
+    type = PiecewiseLinear
+    x = '300 320 340 360 380 400 420 440 460 480 500 520 540 560 580 600 620 640 660 680
+    700 720 740 760 780 800 820 840 860 880 900 920 940 960 980 1000 1020 1040 1060 1080
+    1100 1120 1140 1160 1180 1200 1220 1240 1260 1280 1300 1320 1340 1360 1380 1400 1420 1440 1460 1480 1500'
+    y = '0.15643 0.16354 0.17053 0.17739 0.18415 0.1908 0.19736 0.20382 0.2102 0.2165 0.22272 0.22887 0.23495 0.24096 0.24691 0.2528 0.25864 0.26442 0.27014 0.27582
+    0.28144 0.28702 0.29256 0.29805 0.30349 0.3089 0.31427 0.3196 0.32489 0.33014 0.33536 0.34055 0.3457 0.35082 0.35591 0.36097 0.366 0.371 0.37597 0.38091
+    0.38583 0.39072 0.39558 0.40042 0.40524 0.41003 0.4148 0.41954 0.42426 0.42896 0.43364 0.4383 0.44294 0.44755 0.45215 0.45672 0.46128 0.46582 0.47034 0.47484 0.47932'
+  []
+
+  [h_gapR48]
+    type = PiecewiseLinear
+    x = '300    350    400    450    500    550    600    650    700    750    800    850    900    950    1000'
+    y = '18.55    20.63    22.63    24.55    26.42    28.22    29.98    31.70    33.38    35.03    36.64    38.22    39.78    41.31    42.81'
+  []
+
+  [h_gapR50]
+    type = PiecewiseLinear
+    x = '300    350    400    450    500    550    600    650    700    750    800    850    900    950    1000'
+    y = '24.60    27.36    30.01    32.56    35.03    37.43    39.76    42.04    44.27    46.45    48.58    50.68    52.75    54.78    56.77'
+  []
+
+  [h_gapR52]
+    type = PiecewiseLinear
+    x = '300    350    400    450    500    550    600    650    700    750    800    850    900    950    1000'
+    y = '2.74    3.04    3.34    3.62    3.90    4.16    4.42    4.68    4.92    5.17    5.41    5.64    5.87    6.09    6.32'
+  []
+
+  [beta_fn]
+    type = PiecewiseLinear
+    x = '300  2000'
+    y = '0          0'
+  []
+
+[]
+
+[MaterialProperties]
+
+  [ss-mat]
+    type = HeatConductionMaterialProps
+    k = k304ss
+    Cp = cp304ss
+    rho = 7.93e3
+  []
+
+  [graphite-mat]
+    type = SolidMaterialProps
+    k = kgraphite
+    Cp = cpgraphite
+    rho = rhographite
+  []
+
+  [GC-94F]
+    type = SolidMaterialProps
+    k = kgc94
+    Cp = cpgc94
+    rho = rhogc94
+  []
+
+  [SiC-80]
+    type = SolidMaterialProps
+    k = kSiC80
+    Cp = cpSiC80
+    rho = 2370.0 # OSU-HTTF-TECH-003-R2 Appendix
+  []
+
+  [fluid_props1]
+    type = FunctionFluidProperties
+    rho = rhoHe
+    beta = beta_fn
+    cp = cpHe
+    mu = muHe
+    k = kHe
+    enthalpy = HHe
+  []
```

Comment:
The `FunctionFluidProperties` object is deprecated now, these are moved over directly to `PTFunctionEOS` objects instead.

```suggestion
```